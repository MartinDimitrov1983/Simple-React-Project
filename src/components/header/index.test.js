import React from 'react'
import { render } from '@testing-library/react'

import Header from './index'

test('Snapshot Header', () => {
    const { asFragment } = render(<Header />)
    expect(asFragment()).toMatchSnapshot()
})
