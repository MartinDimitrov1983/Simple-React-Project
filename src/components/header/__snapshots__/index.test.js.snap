// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Header 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <p>
      Search by 
      <i>
        name
      </i>
      /
      <i>
        plan number
      </i>
       or filter by 
      <i>
        region
      </i>
    </p>
    <div
      class="header"
    >
      <div
        class="searchBar"
      >
        <input
          id="search-value"
          name="search-value"
          placeholder="Enter name or plan number"
          type="text"
          value=""
        />
        <button
          class="btn"
          type="button"
        >
          Search
        </button>
      </div>
      <div>
        <select
          class="select"
        >
          <option
            value="none"
          >
            Choose Region
          </option>
        </select>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
